test/integration_tests/configs/suites/quick/avalanche-er.json

```diff
@@ -5,5 +5,6 @@
   "dataset": "cifar10.json",
   "backend": "local",
   "job_name": "class-incremental-mlp-avalanche-er",
-  "expected_accuracy": [0.14100000262260437, 0.6794999837875366]
+  "expected_accuracy_linux": [[0.14100000262260437, 0.6794999837875366]],
```

Comment:
is a 4% difference in performance attributable to platforms?

Reply:
the gap should close if we increase number of epochs from 3 to 50